### ISSUE TITLE
turn on lesson-images.json file warnings on fall2023 branch

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1267]
I'm pretty sure we've already dialed in these warnings and then turned them off, so this issue is just a matter of flipping a switch to turn them back on, but just in case...

- We would like to check that `lesson-images.json` files include all images referenced using `@image` or `@centered-image` in the lesson plan, printable-pages or opt-printable-pages.  
-- _We are ignoring images that only appear in `slides.md` files, as images created only for slides are screenshots of materials that appear elsewhere in a format that can be processed by screen readers_

- We should receive a warning for each `lesson-images.json` file that is either
-- missing an image
-- missing a description, source, or license information for any image 

- `lesson-images.json` files may optionally include captions, but there should be no warnings about captions.

**Comment by ds26gte** [comment]
I think these warnings are already on?  The only difference is that the first type of error (image file missing enirely in the lesson-images.json) has a `**` so that the log reader can learn to discount them, e.g.,

```
WARNING**: Image lessons/computing-needs-all-voices/images/pioneers-header3.png missing from dictionary 
lessons/computing-needs-all-voices/images//lesson-images.json
```

**Comment by flannery-denny** [comment]
I had tried using `./build-pathway` which I thought built everything and got 0 warnings. Is there still a way to build all of the pathways... or are we now limited to only building pyret pathways as a group?

I was able to confirm that `./build-pathway data-science-codap --proglang codap` produces a few `lesson-images.json` warnings.

While I know that I'm detail oriented, I have a hard time believing I didn't miss anything in the algebra or data-science, but maybe?
